lib/point_quest_web/live/quest.ex

```diff
@@ -28,7 +44,7 @@ defmodule PointQuestWeb.QuestLive do
           Phoenix.PubSub.subscribe(PointQuestWeb.PubSub, quest.id)
 
           socket
-          |> assign(quest: quest, users: %{}, form: nil)
+          |> assign(quest: quest, actor: socket.assigns.current_actor, users: %{}, form: nil)
```

Comment:
We could always shorten current actor to just actor? Also as us @current_actor is available in render functions.

Reply:
Oh nice I didn't realize that! I'll get it changed in the morning and re-request your review.

Reply:
I just thought it through (apparently for the first time) and it totally makes sense lmao. I'm grabbing it off the assigns to put it back on the assigns named slightly different. I'm a fool.

Reply:
updated